documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -187,11 +187,13 @@ have completed*.
   - `pull_request.statuses_url`: The url to send build statuses to.
   - `pull_request.head.sha`: The HEAD sha of the pull request branch.
   - `pull_request.base.ref`: The name of the base branch of the pull request.
-  - `pull_request.labels[*].name`: The names of labels attached to the pull
-    request ticket. This will be in the form `["label1", "label2"]. These can
-    be used to trigger different behaviors depending on labels attached to a
-    pull request. In particular, these are used to specify a particular version
-    of Herbert to use in the build.
+  - `pull_request.labels[*].name`:
+    The names of labels attached to the pull request ticket.
+    This will be in the form `["label1", "label2"].
+    These can be used to trigger different behaviors depending on labels
```

Comment:
This comment is misleading. We use the labels for exactly one thing -- the point you make on the next line.

These two lines could be reduced to something more like:
`The inclusion of a label matching the pattern "Herbert_<tagOrBranch>" specifies a particular tag or branch of Herbert to using in the build, otherwise Herbert master is used.`